docs-source/content/userguide/managing-domains/domain-lifecycle/liveness-readiness-probe-customization.md

```diff
@@ -0,0 +1,91 @@
+---
+title: "Liveness and readiness probes customization"
+date: 2019-02-23T20:58:51-05:00
+draft: false
+weight: 6
+description: "This document describes how to customize the liveness and readiness probes for WebLogic Server instance Pods."
+---
+
+This document describes how to customize the liveness and readiness probes for WebLogic Server instance Pods.
+
+#### Contents
+
+* [Liveness probe customization](#liveness-probe-customization)
+* [Readiness probe customization](#readiness-probe-customization)
+
+#### Liveness probe customization
+
+The liveness probe is configured to check that the server is alive by querying the Node Manager process.  The liveness probe is by default configured to check liveness every 15 seconds, and to timeout after 5 seconds.  If a pod fails the liveness probe, Kubernetes will restart that container. 
+
+The liveness probe interval and timeout can be customized using the `livenessProbe` attribute under the `serverPod` element of the domain resource. 
```

Comment:
The liveness probe interval and timeout can be customized -> You can customize the liveness probe interval and timeout